studybuddies/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  private URL = 'http://localhost:3000/api'
+
+  constructor(private http: HttpClient) { }
+
+  signUp(user){
+    return this.http.post<any>(this.URL + '/user/register', user);
+  }
+
+  signIn(user){
+    return this.http.post<any>(this.URL + '/user/login', user);
+  }
+
+  loggedIn() {
+    return !!localStorage.getItem('token');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7163724)
```suggestion
    return !!localStorage.getItem("token");
```